app/views/devise/sessions/new.html.erb

```diff
@@ -0,0 +1,18 @@
+<section class="log-in-section">
+  <%= form_for(resource, as: resource_name, url: session_path(resource_name)) do |f| %>
+    <div class="logIn-inputs">
+      <%= f.text_field :name, autofocus: true, placeholder: "Username", autocomplete: "current-name", class: "name-field" %>
```

Comment:
- [ ] After signing up and confirming my email, when I try to log in I keep getting an invalid email or password error. Kindly note that users should be able to sign in with email and password, not username and password, so you need to update this text field should be changed to email field the attribute should be changed from `:name` to `:email`. The placeholder should also be `Email`, not `Username`.

  ```
  <%= f.email_field :email, autofocus: true, placeholder: "Email", ... %>
  ```

Reply:
Hi @TedLivist , Did you confirm your email before sign in?

Reply:
Yes. It's not confirmation issue. I fixed it myself, that's why you have the suggestion.

Reply:
Thank you for the feedback. I updated the code as required